x-pack/plugins/cloud_security_posture/server/routes/vulnerabilities_dashboard/vulnerabilities_dashboard.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { transformError } from '@kbn/securitysolution-es-utils';
+import type { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+import type { CnvmDashboardData } from '../../../common/types';
+import { VULNERABILITIES_STATS_ROUTE_PATH } from '../../../common/constants';
+import { CspRouter } from '../../types';
+import { getVulnerabilitiesStatistics } from './get_vulnerabilities_statistics';
+
+export interface KeyDocCount<TKey = string> {
+  key: TKey;
+  doc_count: number;
+}
+
+export const defineGetVulnerabilitiesDashboardRoute = (router: CspRouter): void =>
+  router.get(
+    {
+      path: VULNERABILITIES_STATS_ROUTE_PATH,
+      validate: false,
+      options: {
+        tags: ['access:cloud-security-posture-read'],
+      },
+    },
+    async (context, request, response) => {
+      const cspContext = await context.csp;
+
+      try {
+        const esClient = cspContext.esClient.asCurrentUser;
+
+        const query: QueryDslQueryContainer = {
```

Comment:
can we define this query as a constant and share it between the different places?
otherwise, I'm afraid it is going to be hard to maintain it going forward